services/roomService/src/lib/CoveyTownController.ts

```diff
@@ -195,4 +207,17 @@ export default class CoveyTownController {
       nearbyPlayers.includes(listener.getAssociatedPlayer()),
     );
   }
+
+  /**
+   * Informs the correct listener that a direct message has been sent
```

Comment:
i dont think this comment is fully accurate to what the function does. it just retrieves the listeners, it doesn't inform them